tests/test_quantity.py

```diff
@@ -803,6 +827,20 @@ def testing_properties():
     print(
         f"User unit: {x._unit.user_unit}, multiplier: {x._unit.si_factor}, reduced_si_unit: {x._unit.si_unit}, si_value: {x._si_value}"
     )
-
-    c = q.Quantity(1.0, "C")
-    print(float(c))
+    print("")
+
+    c = q.Quantity(1.0, "C^1")
+    m = q.Quantity(1.0, "m")
+    print(f'c = q.Quantity(1.0, "C^1") = {c}')
+    print("")
+    print(f"float(c) = {float(c)}")
```

Comment:
Seems like most of these print statements can be replaced by unit tests plus assertions.  From this code I can't actually tell what each individual operation will actually do.

Reply:
Yes, print statements OK while debugging but not left in for the merge.

Reply:
We are going to remove all code from `if __name__ == "__main__":` onwards later on. It's temporary for now. Thank you.